catalog/ui/search-ui-app/src/main/resources/features.xml

```diff
@@ -20,7 +20,7 @@
     <feature name="catalog-ui" install="auto" version="${project.version}" description="Catalog UI">
         <feature prerequisite="true">camel-http</feature>
         <feature prerequisite="true">search-ui-app</feature>
-        <configfile finalname="/etc/org.codice.ddf.catalog.ui.config.config" override="true">
+        <configfile finalname="/etc/org.codice.ddf.catalog.ui.config.config" override="false">
```

Comment:
The commit message is a little confusing. Something like "Update catalog-ui feature to not override config file in /etc" would be clearer.